### DISCUSSION TITLE
RPU response in CoreELEC

### DISCUSSION THREAD

**Discussion by rome1983** [quietvoid/dovi_tool#228] [Q&A] [answered]
Hi, I'm a CoreELEC user and I had question about the libdovi implementation. When using Minimal EL I'm able to see RPU response with test video vs using 8.1 I get no rpu response. Is that correct behavior?

Test video I used 

https://gofile.me/5QThw/TM2SsIlws

**Comment by quietvoid** [comment, marked as answer]
If your file is FEL then what you might be seeing is the mapping metadata in action.
It is removed by the conversion to profile 8.1, otherwise the image would be inaccurate when the FEL is not used.

**Comment by rome1983** [comment]
Thanks

**Comment by rome1983** [comment]
One more question on another topic. Did you make the bl+el p8 conversion of minions rise of gru with dovibaker? If so how the long process takes? I was trying to figure that out  so I can  gauge if that’s something I wanted to pursue. Thanks

**Comment by quietvoid** [comment]
> Did you make the bl+el p8 conversion of minions rise of gru with dovibaker?

No, I used something else.

I don't recommend doing it if you don't have experience with video encoding.
